shell/platform/fuchsia/flutter/vsync_waiter.cc

```diff
@@ -61,14 +64,56 @@ VsyncWaiter::~VsyncWaiter() {
   ui_latch.Wait();
 }
 
-static fml::TimePoint SnapToNextPhase(fml::TimePoint value,
-                                      fml::TimePoint phase,
-                                      fml::TimeDelta interval) {
-  fml::TimeDelta offset = (phase - value) % interval;
-  if (offset < fml::TimeDelta::Zero()) {
-    offset = offset + interval;
+/// Returns the system time at which the next frame is likely to be presented.
+///
+/// Consider the following timelines, where in both the scenarious the result
+/// will be the same.
+///
+/// Scenario 1:
+/// presentation_interval is 2
```

Comment:
can we clarify here that `presentation_interval` is a time delta, not the number of frames

Reply:
The type of the param is `fml::TimeDelta` as opposed to an integral type, so this should be clear once the reader gets to the method definition.